pallets/automation-time/src/weights.rs

```diff
@@ -349,98 +364,89 @@ impl<T: frame_system::Config> WeightInfo for AutomationWeight<T> {
-	// Storage: AutomationTime Tasks (r:1 w:1)
-	// Storage: AutomationTime ScheduledTasks (r:1 w:1)
+	// Storage: AutomationTime AccountTasks (r:1 w:1)
+	// Storage: AutomationTime ScheduledTasksV3 (r:1 w:1)
 	fn schedule_auto_compound_delegated_stake_task_full() -> Weight {
-		(100_000_000 as Weight)
-		.saturating_add(RocksDbWeight::get().reads(5 as Weight))
-		.saturating_add(RocksDbWeight::get().writes(4 as Weight))
+		(90_076_000 as Weight)
+			.saturating_add(RocksDbWeight::get().reads(5 as Weight))
+			.saturating_add(RocksDbWeight::get().writes(4 as Weight))
 	}
-	// Storage: AutomationTime Tasks (r:1 w:1)
+	// Storage: AutomationTime AccountTasks (r:1 w:1)
 	// Storage: Timestamp Now (r:1 w:0)
 	// Storage: AutomationTime LastTimeSlot (r:1 w:0)
-	// Storage: AutomationTime ScheduledTasks (r:24 w:24)
+	// Storage: AutomationTime ScheduledTasksV3 (r:24 w:24)
 	fn cancel_scheduled_task_full() -> Weight {
-		(1_284_000_000 as Weight)
+		(827_125_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(27 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(25 as Weight))
 	}
-	// Storage: AutomationTime Tasks (r:1 w:1)
+	// Storage: AutomationTime AccountTasks (r:1 w:1)
 	// Storage: Timestamp Now (r:1 w:0)
 	// Storage: AutomationTime LastTimeSlot (r:1 w:0)
-	// Storage: AutomationTime ScheduledTasks (r:1 w:1)
+	// Storage: AutomationTime ScheduledTasksV3 (r:1 w:1)
 	fn force_cancel_scheduled_task() -> Weight {
-		(29_000_000 as Weight)
+		(28_088_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(4 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(2 as Weight))
 	}
-	// Storage: AutomationTime Tasks (r:1 w:1)
+	// Storage: AutomationTime AccountTasks (r:1 w:1)
 	// Storage: Timestamp Now (r:1 w:0)
 	// Storage: AutomationTime LastTimeSlot (r:1 w:0)
-	// Storage: AutomationTime ScheduledTasks (r:24 w:24)
+	// Storage: AutomationTime ScheduledTasksV3 (r:24 w:24)
 	fn force_cancel_scheduled_task_full() -> Weight {
-		(1_274_000_000 as Weight)
+		(823_877_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(27 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(25 as Weight))
 	}
 	fn run_notify_task() -> Weight {
-		(10_000_000 as Weight)
+		(8_779_000 as Weight)
 	}
 	// Storage: System Account (r:2 w:2)
 	fn run_native_transfer_task() -> Weight {
-		(37_000_000 as Weight)
+		(34_814_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(2 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(2 as Weight))
 	}
 	// Storage: XcmpHandler XcmChainCurrencyData (r:1 w:0)
 	// Storage: ParachainInfo ParachainId (r:1 w:0)
 	// Storage: UnknownTokens ConcreteFungibleBalances (r:1 w:0)
 	// Storage: AssetRegistry LocationToAssetId (r:1 w:0)
-	// Storage: System Account (r:2 w:2)
-	// Storage: ParachainSystem HostConfiguration (r:1 w:0)
-	// Storage: ParachainSystem PendingUpwardMessages (r:1 w:1)
 	fn run_xcmp_task() -> Weight {
-		(102_000_000 as Weight)
```

Comment:
same here.  this needs to get reverted back.  not having the asset registry genesis breaks this benchmark for now.